### ISSUE TITLE
config: set default p-queue via ConfigObject 

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1635]
At the moment, you cannot set the optional queue options via CLI. Setting a default via the config options at least allows CLI users to set pqueue options via `config.json`

Another option would be to set `--wps` 3 (3 webhook requests per second)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.12.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.11.8...3.12.0) (2021-05-05)

- build(deps): bump find-up from 4.1.0 to 5.0.0 [`#1633`](https://github.com/open-wa/wa-automate-nodejs/pull/1633)
- build(deps): bump puppeteer from 9.0.0 to 9.1.0 [`#1634`](https://github.com/open-wa/wa-automate-nodejs/pull/1634)
- Release 3.12.0 [`7f948e6`](https://github.com/open-wa/wa-automate-nodejs/commit/7f948e615b06b16a082e78ca39eba138128add2f)
- 🔒  deps: fix 2 moderate vulnerabilities [`378bc57`](https://github.com/open-wa/wa-automate-nodejs/commit/378bc57d6debbd130283f27363030d96c9fc8557)
- ✨implemented `config.pQueueDefault` #1635 [`30142d2`](https://github.com/open-wa/wa-automate-nodejs/commit/30142d290c3a18f2d37b65322b4d308f339e684c)
- 📌  downgrade puppeteer to v9.0.0 [`34b08a2`](https://github.com/open-wa/wa-automate-nodejs/commit/34b08a2e0c32ca2434155115725b593fbaa0f1a3)
- 💥  all `get` request errors now forwarded to end user. Make sure to catch your errors! #1637 [`3e7e2f8`](https://github.com/open-wa/wa-automate-nodejs/commit/3e7e2f81b6d683ed27542d0bef76041cd2b4af76)
- ♻️  refactor some code [`39fffcd`](https://github.com/open-wa/wa-automate-nodejs/commit/39fffcd31e2ef5be4381609a4d283d26b3804310)
